AnnoyancesFilter/sections/popups.txt

```diff
@@ -178,6 +178,7 @@ fandom.com##.banner-notifications-placeholder
 wallstreetmojo.com#$#.pum-overlay[data-popmake*="auto-popup-for-free-ib-courses-videos-desktop"] { display: none!important; }
 wallstreetmojo.com#$#body { overflow: visible!important; }
 goodreads.com##.siteBanner.register
+hubspot.com##div[class^="InitialMessageBubble__StyleWrapper-"]
```

Comment:
this element is inside iframe

Reply:
@Alex-302, rule works for this element on ym side and iframe[src] is the same as hubspot.com